src/sys/unix.rs

```diff
@@ -137,6 +128,15 @@ pub(crate) use libc::{IPV6_ADD_MEMBERSHIP, IPV6_DROP_MEMBERSHIP};
 pub(crate) use libc::{
     IPV6_JOIN_GROUP as IPV6_ADD_MEMBERSHIP, IPV6_LEAVE_GROUP as IPV6_DROP_MEMBERSHIP,
 };
+#[cfg(not(any(
+    target_os = "fuschia",
+    target_os = "illumos",
+    target_os = "netbsd",
+    target_os = "openbsd",
+    target_os = "redox",
+    target_os = "solaris",
+)))]
+pub(crate) use libc::{IP_RECVTOS, IP_RECVTTL};
```

Comment:
Seems like the constant `IP_RECVTTL` is not available on various platform.

Reply:
Reverted this.